shopelectro/images.py

```diff
@@ -0,0 +1,21 @@
+import os
+from django.conf import settings
+
+
+def get_all_images(model, url='products'):
+    """Return all Model images"""
+    images_folder = '{}{}'.format(url + '/', model.id)
+    media_dir = os.path.join(settings.MEDIA_ROOT, images_folder)
+
+    try:
+        images_array = [os.path.normpath(os.path.join(images_folder, file))
+                        for file in os.listdir(media_dir)]
+    except FileNotFoundError:
+        return []
+
+    return list(reversed(images_array))
+
+
+def get_images_without_small(model, url='products'):
+    """Return Model images without small variant"""
+    return [img for img in get_all_images(model, url) if img.find(settings.IMAGE_SMALL_VARIANT) == -1]
```

Comment:
`filter(lambda img: img.find(settings.IMAGE_SMALL_VARIANT) not -1, get_all_images(model, url))`